src/main/java/org/interledger/setup/spsp/model/Payee.java

```diff
@@ -0,0 +1,13 @@
+package org.interledger.setup.spsp.model;
+
+import java.net.URI;
+
+public interface Payee extends SpspReceiver {
+  
+  public String getName();
+
+  //TODO Should we return a java.awt.Image?

```

Comment:
no, i would definitely leave it as a url - there's no point in fetching that data unless someone explicitly asks for it.